dashboard/app/controllers/maker_controller.rb

```diff
@@ -15,14 +15,12 @@ def home
     }
   end
 
-  # Returns which devices script version to show:
-  #   Assigned script should take precedence - show most recent version that's been assigned.
-  #   Otherwise, show the most recent version with progress.
-  #   If none of the above applies, default to most recent.
+  # TODO: deprecate this page which is no longer active since the Maker app was deprecated - https://support.code.org/hc/en-us/articles/11304760762125-The-Maker-App-and-Chrome-Serial-Extension-Are-Now-Deprecated.
```

Comment:
Very long line; can we break it up?

Also, what's stopping us from deprecating the page now?

Reply:
Just wanting to get in a quick fix so page can actually load.
Just needing to get approval from product and want to be a little bit more careful about removing old code/routes.

Reply:
>Very long line; can we break it up?

Sure! Just removed the long link from comment.